src/btree/bt_vrfy.c

```diff
@@ -281,6 +285,30 @@ __verify_key_hs(WT_SESSION_IMPL *session, WT_ITEM *key, WT_VSTUFF *vs)
 #else
         WT_UNUSED(vs);
 #endif
+
+        /* Perform Verification of data continuity. */
```

Comment:
I'd prefer something like "verify that the current record's stop time pair doesn't overlap the start time pair of its successor".